osquery/devtools/shell.cpp

```diff
@@ -56,6 +56,7 @@ namespace osquery {
 /// Define flags used by the shell. They are parsed by the drop-in shell.
 SHELL_FLAG(bool, csv, false, "Set output mode to 'csv'");
 SHELL_FLAG(bool, json, false, "Set output mode to 'json'");
+SHELL_FLAG(bool, pack_single_json, false, "Set output mode to 'pack_single_json'");
```

Comment:
nitpick, this is a long flag, can `--mjson` stand for "merged JSON" or `--sjson` for "single JSON" work?

Reply:
In any case I think the flag description needs a detail about what this does, the others are more obvious, hehe!

Reply:
hey @theopolis ,
I'm happy with any long flag name that makes the most sense.  I'll refactor to using --mjson on next commit when i add the more detailed description.